README.md

```diff
@@ -563,7 +563,7 @@ Flags:
       --amount string        Deposit amount
       --contract string      ERC20 contract address
       --decimals uint        ERC20 token decimals
-      --destination string   Where tokens should be minted. Defaults to TX sender
+      --recipient string     Recipient address
```

Comment:
I actually prefer `--destination` but changed it to be consistent with the [ERC721.Mint](https://github.com/ChainSafe/chainbridge-core/blob/waymobetta/cli-validation-tests/chains/evm/cli/erc721/mint.go#L58). 

Open to discussion!